lib/metrics/default-metrics.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+const ClientMetrics = require('./client-metrics');
+const EventEmitter = require('events');
+
+/**
+ * A default implementation of [ClientMetrics]{@link module:metrics~ClientMetrics} that exposes the driver events as
+ * Node.js events.
+ * <p>
+ *   An instance of [DefaultMetrics]{@link module:metrics~DefaultMetrics} is configured by default in the client,
+ *   you can access this instance using [Client#metrics]{@link Client#metrics} property.
+ * </p>
+ * @implements {module:metrics~ClientMetrics}
+ * @alias module:metrics~DefaultMetrics
+ * @example <caption>Listening to events emitted</caption>
+ * defaultMetrics.errors.on('increment', err => totalErrors++);
+ * defaultMetrics.errors.clientTimeout.on('increment', () => clientTimeoutErrors++);
+ * defaultMetrics.speculativeRetries.on('increment', () => specExecsCount++);
+ * defaultMetrics.successfulResponses.on('increment', latency => specExecsCount++);
```

Comment:
This should probably be `responseCount++` or maybe `totalLatencySeconds += latency[0]`

Reply:
🤦